pkgs/by-name/tu/tuist/package.nix

```diff
@@ -0,0 +1,52 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchurl,
+  unzip,
+  nix-update-script,
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "tuist";
+  version = "4.38.2";
+
+  src = fetchurl {
+    url = "https://github.com/tuist/tuist/releases/download/${finalAttrs.version}/tuist.zip";
+    hash = "sha256-FK9F0Y3p04NOoy1Mnlcvimm/LGA5Y+lQ9P679SNNOzA=";
+  };
+
+  dontUnpack = true;
+  dontPatch = true;
+  dontConfigure = true;
+  dontBuild = true;
+  dontFixup = true;
+
+  nativeBuildInputs = [ unzip ];
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/opt/tuist/
+    unzip $src -d $out/opt/tuist/
+
+    mkdir -p $out/bin/
+    ln -s $out/opt/tuist/tuist $out/bin/tuist
+
+    runHook postInstall
+  '';
+
+  passthru = {
+    updateScript = nix-update-script { };
+  };
+
+  meta = {
+    description = "Command line tool that leverages project generation to abstract intricacies of Xcode projects";
```

Comment:
I realise this is taken direct from their page, but it reads a lot like marketing copy and it isn't very clear what it means. Is there a better short description around?

Reply:
Took the one from the [v3 README](https://github.com/tuist/tuist/tree/3.42.3?tab=readme-ov-file#whats-tuist-)